pkg/cache/partition_info.go

```diff
@@ -93,6 +94,18 @@ func NewPartitionInfo(partition configs.PartitionConfig, rmId string, info *Clus
     // TODO get the resolver from the config
     p.userGroupCache = security.GetUserGroupCache("")
 
+    // TODO all policies are looped here, and considers
+    // only scheduler policy. Need some more cleaner interface here.
+    for _, policy := range partition.GlobalPolicies {
+        switch policy.Policy {
+        case configs.SchedulingBinPackingPolicy:
+            p.globalSchedulingPolicy = NewSchedulingPolicy(policy)
+            log.Logger().Info("SchedulingBinPackingPolicy policy is set.")
+        default:
+            log.Logger().Info("No default scheduling policy is set.")
+        }
+    }
+
```

Comment:
We need to define what is "default scheduling policy", to me it is ambiguous to say there's no "default scheduling policy". There should be always one policy, such as "round-robin".

Reply:
+1

Reply:
yes. agree.